yabt/buildcontext.py

```diff
@@ -524,4 +561,11 @@ def build_in_pool(seq):
                     self.conf.flavor, self.conf.jobs)
         # main pass: build rest of the graph
         build_in_pool(self.target_iter())
-        logger.info('Finished building target graph successfully')
+        if self.failed_nodes:
+            msg = ('Finished building target graph with fails: \n{}\n'
+                   'which caused the following to skip: \n{}'.format(
+                   self.failed_nodes, self.skipped_nodes))
+            print(msg)
```

Comment:
@itamaro Can't use "fatal" here cause we're not in an exception context. Which fails only on python 3.4
WDYT?